documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,134 @@
+
+
+**Q:** Is the Administration Server required? Node Manager?
+
+**A:** Certification of both WebLogic running in containers and WebLogic in Kubernetes consists of a WebLogic domain with the Administration Server. The operator configures and runs Node Managers for you within a domain's pods - you don't need to configure them yourself - so their presence is largely transparent.
+
+***
+
+#### Communications
+
+**Q:** How is location transparency achieved and the communication between WLS instances handled?
+
+**A:** Inside the Kubernetes cluster, the operator generates a Kubernetes ClusterIP service for each WebLogic Server pod called `DOMAINUID-WEBLOGICSERVERNAME` and for each WebLogic cluster called `DOMAINUID-cluster-CLUSTERNAME`. The operator also overrides your WebLogic network listen address configuration to reflect the service names so that you don't need to do this. The services act as DNS names, and allow the pods with WebLogic Servers to move to different nodes in the Kubernetes cluster and continue communicating with other servers.  
+
+**Q:** How is communication from outside the Kubernetes cluster handled?
+
+**A:** HTTP communication to your applications from locations outside the cluster is typically accomplished by deploying a load balancer that redirects traffic to your domain's Kubernetes services (the operator automatically deploys these services for you). For example, see the Quick Start, [Install the operator and ingress controller]({{< relref "/quickstart/install.md" >}}).
+
+RMI communication with locations outside of the Kubernetes cluster is typically accomplished by deploying a load balancer or using Kubernetes Node Ports. See [External WebLogic clients]({{< relref "/faq/external-clients.md" >}}).
+
+* Access to arbitrary HTTP applications through a load balancer.  See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
+* Access the WebLogic Server Administration Console through a load balancer.  See the [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample.
+* Access the WebLogic Server Administration Console through a Kubernetes NodePort service.  Run `$ kubectl explain domain.spec.adminServer.adminService.channels`.
+* WebLogic Server Remote Console communication is accomplished by using a load balancer. See [Use the Remote Console]({{< relref "/userguide/managing-domains/accessing-the-domain/admin-console.md" >}}).
```

Comment:
Can we organize the answer a little differently? For example:

```
**Q:** How is communication from outside the Kubernetes cluster handled?

**A:** 

* _HTTP communication to your applications from locations outside the cluster:_  This is typically accomplished by deploying a load balancer that redirects traffic to your domain's Kubernetes services (the operator automatically deploys these services for you).  See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
For example, see the Quick Start, [Install the operator and ingress controller]({{< relref "/quickstart/install.md" >}}).
* _JMS, EJB, and other types of RMI communication with locations outside of the Kubernetes cluster:_  This is typically accomplished by tunneling the RMI traffic over HTTP through a load balancer or, less commonly accomplished by using T3 or T3S directly with Kubernetes Node Ports. See [External WebLogic clients]({{< relref "/faq/external-clients.md" >}}).
* _Access the WebLogic Server Administration Console_:  This can be done through a load balancer, see the [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample.  Or this can be done through a Kubernetes NodePort service, run `$ kubectl explain domain.spec.adminServer.adminService.channels`.
* _Access the WebLogic Server Remote Console_: This can be done using a load balancer or Kubernetes NodePort service. See [Use the Remote Console]({{< relref "/userguide/managing-domains/accessing-the-domain/admin-console.md" >}}).
```

Reply:
Yes, 100% better. Done.